user_manual/groupware/contacts.rst

```diff
@@ -122,8 +122,65 @@ to create new address books, simply by specifying an address books name:
 .. figure:: ../images/contact_manageaddressbook.png
 	:alt: Add address book in the contacts settings
 
-The Contacts settings is also where you can share, export and delete address
-books. You will find the CardDAV URLs there.
+The Contacts settings is also where you can share, export and delete addressbooks. You will find the CardDAV URLs there.
 
 See :doc:`index` for more details about syncing your address books
 with iOS, macOS, Thunderbird and other CardDAV clients.
+
+
+Circles
+-------
+
+Informal collaboration takes place within organizations: an event to organize for a few weeks, a short ideation session between members from differents entities, workshops, a place to joke around and support team building, or simply in very organic organizations where formal structure is kept to a minimum.
+
+For all these reasons, Nextcloud supports Circles, a feature embedded in the Contacts app, where every user is able to create its own circle, a user-defined aggregate of users. Circles can be used later on to share files and folders, added to Talk conversations, like a regular group.
+
+.. figure:: ../images/circle.png
+	:alt: Circle in the Contacts app left menu
+
+Create a circle
+~~~~~~~~~~~~~~~
+
+In the left menu, click on the + next to Circles.
+Set a circle name
+Landing on the circle configuration screen, you can:
+- add users to your circle
```

Comment:
I would write "add _members_ to your circle" as a member can be a local account, a remote mail address, another circle, ...